### DISCUSSION TITLE
My Github feed is empty

### DISCUSSION THREAD

**Discussion by cwegener** [community/community#66196] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

All my daily feed content has disappeared when I logged in to Github this morning.

Anyone has experienced the same thing? And has an idea of where the feed updates are located?

**Comment by DNin01** [comment]
It's probably because GitHub just [updated the feed](https://github.blog/changelog/2023-09-06-updates-to-your-github-feed/).

Any chance that it has to do with the filters? (Try clicking the "Filter" button.)

**Reply by cwegener** [reply]
Nope. All gone. None of the activity of the people I follow on Github is available in the Github homepage feed anymore.

I'm now manually creating the necessary RSS feeds for each of the people I follow to restore this functionality.

It's a painful process to go through, but it's worth it.

I think that this example shows that we need a new decentralized platform for coding related status updates. Something like ActivityPub for code repositories.

**Reply by cwegener** [reply]
To clarify, I was referring to this feed: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-user-account-settings/about-your-personal-dashboard#following-feed

It has disappeared for me, but using the link that go-spider shared made the "Following feed" available directly through that link (it's just not available for me in the Github homepage / dashboard)

**Comment by go-spider** [comment, marked as answer]
https://github.com/dashboard-feed

**Reply by cwegener** [reply]
You are a legend. Thanks a million. It will buy me some time for migrating to RSS feeds .... until Microsoft takes away RSS ... :joy:
